### PR TITLE
Update the applying flow of proposal to 2023

### PR DESCRIPTION
**Pull request by josix**
## Types of changes
**Thanks for sending a pull request! Please fill in the following content to let us know better about this change.**
Please put an `x` in the box that applies

- [x] **New feature**

## Description
**Describe what the change is**
* Remove pre_recorded_policy from event
* Update correct PERFER_TIME_CHOICES options

### CODE REVIEW

File: src/locale/zh_Hant/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -4669,3 +4889,31 @@ msgstr ""
 #: users/views.py:129
 msgid "Password reset successful. You can now login."
 msgstr "密碼重設成功。您現在即可登入。"
+
```
Comment (mattwang44) [inline src/locale/zh_Hant/LC_MESSAGES/django.po:4892]: let's delete these outdated line

File: src/locale/en_US/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -4799,3 +5017,33 @@ msgstr ""
 #: users/views.py:129
 msgid "Password reset successful. You can now login."
 msgstr "Password reset successful. You can now login."
+
```
Comment (mattwang44) [inline src/locale/en_US/LC_MESSAGES/django.po:5020]: ditto

**Review by mattwang44 — APPROVED** [review]
nit:

1. `.po` 裡面有一些 room name 可以直接填上原文
2. （雖然我們沒有規範，但是比較好的 practice 是）migration files 可以 rename. e.g. `0056_auto_20230108_2145.py` -> `0056_update_2023_policies.py`

File: src/locale/zh_Hant/LC_MESSAGES/django.po (hunk 1 above)
Comment (josix) [reply src/locale/zh_Hant/LC_MESSAGES/django.po:4892]: Thanks, good catch!

**Pull request merged**